ui/app/rif/pages/common-style-pages.css

```diff
@@ -15,10 +15,11 @@
 .hand-over {
   cursor: pointer;
 }
+
 .body {
   width: 100%;
-  overflow: scroll;
-  height: 26.5em;
+  overflow-y: auto;
+  height: 352px;
```

Comment:
Why is this fixed in pixels, can we change it to be not a fixed value?

Reply:
I dont really remember this change, in this PR i didn't touch any css... maybe because i merge it with other branches

Reply:
Remember that this PR has changes of @fmelo